src/Evolution/Systems/CurvedScalarWave/Worldtube/Tags.hpp

```diff
@@ -65,6 +67,51 @@ struct ExpansionOrder {
  * \brief Tags related to the worldtube
  */
 namespace Tags {
+/*!
+ * \brief Dummy tag that throws an error if the input file does not describe a
+ * circular orbit.
+ */
+template <size_t Dim>
+struct CheckCircularOrbit : db::SimpleTag {
+  using type = bool;
+  using option_tags = tmpl::list<domain::OptionTags::DomainCreator<Dim>,
+                                 OptionTags::ExcisionSphere>;
+  static constexpr bool pass_metavariables = false;
+  static bool create_from_options(
+      const std::unique_ptr<::DomainCreator<Dim>>& domain_creator,
+      const std::string& excision_sphere_name) {
+    const auto domain = domain_creator->create_domain();
+    const auto& excision_spheres = domain.excision_spheres();
+    const auto& excision_sphere = excision_spheres.at(excision_sphere_name);
+    const double orbital_radius = get<0>(excision_sphere.center());
+    const auto& functions_of_time = domain_creator->functions_of_time();
+    if (not functions_of_time.count("Rotation")) {
+      ERROR("Expected functions of time to contain 'Rotation'.");
+    }
+    // dynamic cast to access `angle_func_and_deriv` method
+    const auto* rotation_function_of_time =
+        dynamic_cast<domain::FunctionsOfTime::QuaternionFunctionOfTime<3>*>(
+            &*functions_of_time.at("Rotation"));
+    ASSERT(rotation_function_of_time,
+           "Failed dynamic cast to QuaternionFunctionOfTime.");
+    const auto angular_velocity =
+        rotation_function_of_time->angle_func_and_deriv(0.).at(1);
+    if (equal_within_roundoff(orbital_radius, 0.)) {
+      ERROR("The orbital radius was set to 0.");
+    }
+    if (not(equal_within_roundoff(angular_velocity.at(0), 0.) and
+            equal_within_roundoff(angular_velocity.at(1), 0.) and
+            equal_within_roundoff(angular_velocity.at(2),
+                                  pow(orbital_radius, -1.5)))) {
```

Comment:
`equal_within_roundoff` can take iterables and `angular_velocity` is a DataVector so I think this can just be

```cpp
if (not equal_within_roundoff(angular_velocity, DataVector{0.0, 0.0, pow(orbital_radius, -1.5)})) {
```